packages/docs/src/components/MedplumCodeBlock.tsx

```diff
@@ -0,0 +1,103 @@
+import CodeBlock, { Props } from '@theme/CodeBlock';
+import * as React from 'react';
+
+const BLOCK_START_PATTERN = new RegExp('^//\\s*start-block\\s+([A-Za-z_-]*)', '');
+// const BLOCK_START_PATTERN = /^\/\/\s*start-block\s+([A-Za-z_-]*)/;
+const BLOCK_END_PATTERN = new RegExp('^//\\s*end-block\\s+([A-Za-z_-]*)', '');
+
+interface MedplumCodeBlockProps extends Props {
+  selectLines?: number[][];
+  selectBlocks?: string[] | string;
+}
+
+export default function MedplumCodeBlock({
+  children,
+  selectLines,
+  selectBlocks,
+  ...props
+}: MedplumCodeBlockProps): JSX.Element {
+  let code = children as string;
```

Comment:
I'm kinda surprised this worked, because this is not how `children` normally works in React.

If it is required to be a `string`, then it should probably be a named prop (i.e., `code: string` or whatever)